lib/SIL/SILDeclRef.cpp

```diff
@@ -842,7 +883,9 @@ SILDeclRef SILDeclRef::getOverridden() const {
   if (!overridden)
     return SILDeclRef();
 
-  return SILDeclRef(overridden, kind, isCurried);
+  // SWIFT_ENABLE_TENSORFLOW
+  return SILDeclRef(overridden, kind, isCurried, isForeign,
+                    autoDiffAssociatedFunctionIdentifier);
 }
```

Comment:
```suggestion
// SWIFT_ENABLE_TENSORFLOW END
}
```

Reply:
I didn't feel this was necessary, since the change was two lines.
I've been using `SWIFT_ENABLE_TENSORFLOW END` to mark sections where the end point isn't so clear.

Reply:
Fair!